app/src/main/java/com/genymobile/pr/ui/PullRequestDetailsDialogFragment.java

```diff
@@ -64,30 +84,31 @@ private void getIssue() {
     @Subscribe
     @SuppressWarnings("unused")
     public void onIssueRetrieved(IssueRetrievedEvent event) {
+        int defaultSpacing = dpToPx(DEFAULT_SPACING_PX);
         LinearLayout labelsView = (LinearLayout) PullRequestDetailsDialogFragment.this.getDialog().findViewById(R.id.labels);
         LayoutParams layoutParams = new LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT);
-        layoutParams.setMargins(DEFAULT_SPACING, DEFAULT_SPACING, DEFAULT_SPACING, DEFAULT_SPACING);
+        layoutParams.setMargins(defaultSpacing, defaultSpacing, defaultSpacing, defaultSpacing);
         Issue issue = event.getIssue();
         for (Label label : issue.getLabels()) {
             addLabel(labelsView, label, layoutParams);
         }
     }
 
+    private int dpToPx(int dp) {
+        DisplayMetrics displayMetrics = getContext().getResources().getDisplayMetrics();
+        return Math.round(dp * (displayMetrics.xdpi / DisplayMetrics.DENSITY_DEFAULT));
```

Comment:
why not http://stackoverflow.com/a/6327095 ? seems shorter and may not need a separate method